app/middleware/Redirect.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace GanttDashboard\App\Middleware;
+
+use Phalcon\Application;
+use Phalcon\Events\Event;
+use GanttDashboard\App\Services\Authentication as AuthenticationService;
+
+/**
+ * Redirect
+ *
+ * Checks the request and redirects the user somewhere else if needed
+ */
+class Redirect
```

Comment:
i believe that this is an authentication middleware that checks if a user is authenticated and if not it gets redirected